react-example/src/Components/LayoutLogin/LayoutLogin.less

```diff
@@ -0,0 +1,46 @@
+main{
```

Comment:
We like to leave an space between the selector and the opening curly brace `{`.

Insead of `main{` use  `main {`.

Please fix this on all of the less files